sig-api-machinery/README.md

```diff
@@ -24,6 +24,85 @@ Covers all aspects of API server, API registration and discovery, generic API CR
 * [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-api-machinery)
 * [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/sig%2Fapi-machinery)
 
+## Subprojects
+
+The following subprojects are owned by this SIG:
+- **api**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/api/master/OWNERS
+- **apiextensions-apiserver**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/apiextensions-apiserver/master/OWNERS
+- **apimachinery**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/apimachinery/master/OWNERS
+- **apiserver**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/apiserver/master/OWNERS
+- **apiserver-builder**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-incubator/apiserver-builder/master/OWNERS
+- **client-go**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/client-go/master/OWNERS
+- **client-python**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-incubator/client-python/master/OWNERS
+- **code-generator**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/code-generator/master/OWNERS
+- **community**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/community/master/OWNERS
+- **csharp**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/csharp/master/OWNERS
+- **gen**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/gen/master/OWNERS
+- **gengo**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/gengo/master/OWNERS
+- **go**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/go/master/OWNERS
+- **go-base**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/go-base/master/OWNERS
+- **haskell**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/haskell/master/OWNERS
+- **java**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/java/master/OWNERS
+- **javascript**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/javascript/master/OWNERS
+- **kube-aggregator**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/kube-aggregator/master/OWNERS
+- **kube-openapi**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes/kube-openapi/master/OWNERS
+- **python-base**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/python-base/master/OWNERS
+- **ruby**
+  - Owners:
+    - https://raw.githubusercontent.com/kubernetes-client/ruby/master/OWNERS
+- **sample-apiserver**
```

Comment:
sample-apiserver and sample-controller would make sense in the apiserver subproject.